test/org/sagebionetworks/bridge/hibernate/HibernateAccountDaoTest.java

```diff
@@ -72,19 +60,16 @@
     private static final String ACCOUNT_ID = "account-id";
     private static final DateTime CREATED_ON = DateTime.parse("2017-05-19T11:03:50.224-0700");
     private static final String DUMMY_PASSWORD = "Aa!Aa!Aa!Aa!1";
-    private static final String DUMMY_PASSWORD_HASH = "dummy-password-hash";
-    private static final String DUMMY_REAUTH_TOKEN_HASH = "dummy-reauth-token-hash";
     private static final String EMAIL = "eggplant@example.com";
     private static final Phone PHONE = TestConstants.PHONE;
     private static final Phone OTHER_PHONE = new Phone("+12065881469", "US");
     private static final String OTHER_EMAIL = "other-email@example.com";
     private static final String HEALTH_CODE = "health-code";
-    private static final long MOCK_NOW_MILLIS = DateTime.parse("2017-05-19T14:45:27.593-0700").getMillis();
+    private static final DateTime MOCK_NOW_MILLIS = DateTime.parse("2017-05-19T14:45:27.593-0700");
```

Comment:
Nitpick: This is onw a date-time rather than epoch milliseconds, and should thus be renamed accordingly.

Reply:
Changed name.